### DISCUSSION TITLE
Activity Feed not working

### DISCUSSION THREAD

**Discussion by Flone-dnb** [community/community#22769] [New to GitHub]
<p>It’s been 4 months already and I’m still getting this error (There was an error in loading the activity feed). Is there a solution?</p>

**Comment by akshay-vinod** [comment]
<p>Same here,</p>
<p>** “There was an error in loading the activity feed”** this is what  I getting</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>We’re aware that there is an issue affecting many users and our engineers are still working on it.</p>
<p>For some people, it has helped if we clear your activity feed from our end so that it starts fresh. If you’d like me to try that, let me know.</p>

**Reply by Apollo-Blaze** [reply]
> 👋 Welcome!
> 
> We’re aware that there is an issue affecting many users and our engineers are still working on it.
> 
> For some people, it has helped if we clear your activity feed from our end so that it starts fresh. If you’d like me to try that, let me know.

I have the same problem can you do it for me

**Reply by jonathanlaang** [reply]
Have the same issue too

**Reply by NicoCaldo** [reply]
The feed is still not working https://github.com/orgs/community/discussions/143676#discussioncomment-11539109

**Comment by Flone-dnb** [comment]
<p>Yes, let’s try that, you can clear my activity feed.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>That’s done! Please give it a few hours and see if it starts working.</p>

**Comment by Flone-dnb** [comment]
<p>Yep, it’s already working. Thanks.</p>

**Comment by Rajeshwari2024** [comment]
Try clearing your browser cache, using incognito mode, and checking [GitHub Status](https://www.githubstatus.com/). If the issue persists, contact [GitHub Support](https://support.github.com/).

**Comment by ParasSharmaCMS** [comment]
<img width="959" height="473" alt="image" src="https://github.com/user-attachments/assets/d2a1dad2-faf9-4e57-94cb-609b081b0f89" />

I have been working on a project and still not able to figure out why dots are not giving me activity dots its been weeks that I'm pushing code
